src/components/Forms/editUserForm.tsx

```diff
@@ -30,20 +41,24 @@ const EditUserForm = (props: userFormType) => {
 	const dispatch = useAppDispatch();
 	const { showErrorMessage, showSuccessMessage } = useToast();
 	const id = props.id;
+	const isAccountActive = props.useR[0]?.isActive;
 	const { accessToken } = useToken();
 	const navigate = useNavigate();
-	const url = window.location.pathname;
+
+	const { enable, reason } = useAppSelector((state) => state.enableAccount) as {
+		enable: string;
+		reason: string;
+	};
 
 	useEffect(() => {
 		if (!roles) {
 			dispatch(getRoles()).unwrap();
 		}
-	}, [dispatch, roles]);
+		dispatch(setEnable(isAccountActive ? 'true' : 'false'));
+	}, [dispatch, isAccountActive]);
 
 	const handleNavigation = () => {
-		if (url) {
-			navigate(-1);
-		}
+		navigate(-1);
```

Comment:
what are you real trying to here? navigate(-1); ? provide more clarifications on this code?